src/components/docs-version-switcher/docs-version-switcher.module.css

```diff
@@ -1,114 +1,36 @@
-.root {
-  --padding-left-right: 24px;
-
-  background-color: white;
-  width: 100%;
-
-  /* custom elevation; no HDS token at the time of building */
-  box-shadow: 0 -2px 2px rgba(101, 106, 118, 0.1),
-    0 -8px 16px -10px rgba(101, 106, 118, 0.1);
-
-  &:hover {
-    /* custom elevation; no HDS token at the time of building */
-    box-shadow: 0 -2px 4px rgba(101, 106, 118, 0.15),
-      0 -16px 18px -8px rgba(101, 106, 118, 0.1);
-  }
-}
-
-.root-expanded {
-  /* custom elevation; no HDS token at the time of building */
-  box-shadow: 0 -2px 4px rgba(101, 106, 118, 0.15),
-    0 -16px 18px -8px rgba(101, 106, 118, 0.1);
-}
-
-.activator {
-  background-color: white;
-  border: none;
-  color: var(--token-color-foreground-faint);
-  padding-bottom: 12px;
-  padding-left: var(--padding-left-right);
-  padding-right: var(--padding-left-right);
-  padding-top: 12px;
-
-  &:focus {
-    outline: transparent;
-  }
-
-  &:focus-visible {
-    & .activatorTextWrapper {
-      box-shadow: var(--token-focus-ring-action-box-shadow);
-    }
-  }
-}
-
-.activatorTextWrapper {
-  align-items: center;
-  border-radius: 5px;
-  display: flex;
-  gap: 6px;
-  padding: 8px;
-  width: 100%;
-  position: relative;
-}
-
-.activatorText {
-  flex-grow: 1;
-  text-align: left;
-}
-
-.activatorIcon {
-  flex-shrink: 0;
-  display: flex;
-}
-
-.content {
-  padding-left: calc(var(--padding-left-right) - 3px);
-  padding-right: calc(var(--padding-left-right) - 3px);
-}
-
-.optionList {
-  border-top: 1px solid var(--token-color-border-primary);
-  padding-top: 12px;
-  padding-bottom: 12px;
-  max-height: 200px;
-  overflow-x: hidden;
-  overflow-y: auto;
-}
-
-.optionListItem {
-  padding-left: 3px;
-  padding-right: 3px;
-}
-
-.optionLink {
-  /* composition */
-  composes: g-focus-ring-from-box-shadow from global;
-
-  /* css properties */
-  background-color: white;
-  border-radius: 5px;
-  color: var(--token-color-foreground-faint);
-  display: block;
-  font-family: var(--token-typography-body-200-font-family);
-  font-size: var(--token-typography-body-200-font-size);
-  line-height: var(--token-typography-body-200-line-height);
-  padding: 8px;
-  position: relative;
-  text-decoration: none;
-
-  &[aria-current='page'] {
-    background-color: var(--token-color-palette-neutral-200);
-    color: var(--token-color-foreground-strong);
-  }
-
-  &:hover {
-    background-color: var(--token-color-palette-neutral-100);
-    color: var(--token-color-foreground-strong);
-    text-decoration: none;
-  }
-
-  &:focus-visible {
-    color: var(--token-color-foreground-strong);
-    z-index: 1;
-  }
+.docsVersionSwitcher {
+	@media (--dev-dot-mobile) {
+		width: 100%;
+
+		& button {
+			width: 100%;
+			justify-content: space-between;
+		}
+
+		& > div {
+			width: 100%;
+		}
+
+		& ul {
+			width: 100%;
+			max-width: 100%;
+		}
```

Comment:
suggestion / stylistic: I think having explicit classnames here can be helpful just to more clearly correlate with the underlying elements, e.g. when I go look at the component source I can see immediately where the styles are intended to be applied.

Reply:
Agreed, however the styles here are editing another component (`<DropdownDisclosure />`). We could add comments to the css to address this concern. Alternatively, we could add `className` props to the main component, but this may be overkill. What do you think?

Reply:
Ah word, this makes sense! Maybe we can add a comment here explaining what it's for (making the dropdown full width on mobile)? It feels like this might be something we want to bake into the component at some point with a prop 💭

Reply:
Definitely!

Reply:
[Added](https://github.com/hashicorp/dev-portal/pull/945/files#diff-d6b6fc60cb7fc4bc24f617ad293058ae0b7f76ced0f7605fa3d4db08dbbc69a0R3)!